probcal/experiments/ood.py

```diff
@@ -1,19 +1,39 @@
-from probcal.utils.experiment_utils import get_model, get_datamodule
+import matplotlib.pyplot as plt
+
 
+from probcal.utils.experiment_utils import get_model, get_datamodule
+from probcal.utils.configs import TestConfig
+from probcal.utils.img_utils import denormalize
 from probcal.enums import DatasetType, ImageDatasetName, HeadType
 
+
+
+# build dataset and data loader
 datamodule = get_datamodule(
         DatasetType.IMAGE,
-        ImageDatasetName.COCO_PEOPLE,
+        ImageDatasetName.OOD_COCO_PEOPLE,
         1,
         num_workers=0
     )
-datamodule.prepare_data()
-datamodule.setup("test")
-
+datamodule.setup(stage="test")
 test_loader = datamodule.test_dataloader()
 
+# instantiate model
+model_cfg = TestConfig.from_yaml(f"configs/test/coco_gaussian_cfg.yaml")
```

Comment:
The setup I've been working with is a monolith config that just stays "attached" to model weights. For an example of how I used the train config's information to evaluate a model for the DDPN experiments, you can see `deep_uncertainty/evaluation/eval_model.py`. The configs should be saved in/near the GCS buckets with the model weights. I can help you track them down if you'd like. Then again, if this is all unnatural and the `TestConfig` setup woroks better for you, it's not a huge adjustment. Might be better since we haven't written any real evaluation code in this repo yet

Reply:
Decided to split into TrainConfig/TestConfig